themes/round-hp/assets/js/components/pokemon.js

```diff
@@ -0,0 +1,112 @@
+Vue.component("Pokemon", {
+  template: `
+    <div 
+      :class="{'pkmn': true, 'pkmn--fainted': isFainted, 'pkmn--empty': isEmptySlot}" 
+      :style="{'background': getBackgroundColor }"
+    >
+      <div v-if="typeof pokemon == 'object'" class="pkmn__wrapper">
```

Comment:
if it's empty, this conditional won't matter

Reply:
Can you elaborate? I embed `<Pokemon>` no matter if there is a member of the party or not, then Pokemon takes care of rendering the right styles (border is handled different for empty slot for example), removing the condition breaks the theme if there are any empty spots on the team